core/frontend/src/components/vehiclesetup/configuration/failsafes/Failsafes.vue

```diff
@@ -0,0 +1,229 @@
+
+          params: [
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer',
+              name: 'FS_GCS_TIMEOUT',
+              optional: true,
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_GCS_ENABLE',
+              replacementDescription: '',
+            },
+          ],
+        },
+        {
+          name: 'Pilot Input Loss',
+          generalDescription: 'Triggers when the vehicle does not receive any pilot input for a given '
+          + 'amount of time.',
+          image: (await import('@/assets/img/configuration/failsafes/pilot-input.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer-outline',
+              name: 'FS_PILOT_TIMEOUT',
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PILOT_INPUT',
+            },
+          ],
+        },
+        {
+          // TODO: move actual pin configuration to its own setup page
+          name: 'Leak Detection',
+          generalDescription: 'Triggers when a leak is detected. We recommend keeping control electronics dry.',
+          image: (await import('@/assets/img/configuration/failsafes/leak.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Leak probe 1 type',
+              name: 'LEAK1_TYPE',
+              optional: true,
+            },
+            {
+              replacementTitle: 'Leak 1 pin',
+              icon: 'mdi-pin',
+              name: 'LEAK1_PIN',
+            },
+            {
+              replacementTitle: 'Leak 1 logic level when dry',
+              name: 'LEAK1_LOGIC',
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_LEAK_ENABLE',
+            },
+          ],
+        },
+        {
+          name: 'Excess Internal Pressure',
+          generalDescription: 'Triggers when the internal pressure is too high. This may help to detect a leak, '
+          + 'and to avoid rapid unplanned disassembly.',
+          image: (await import('@/assets/img/configuration/failsafes/pressure.svg')).default as string,
+          params: [
+
+            {
+              replacementTitle: 'Maximum internal pressure',
+              icon: 'mdi-gauge-full',
+              name: 'FS_PRESS_MAX',
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PRESS_ENABLE',
+            },
+          ],
+        },
+        {
+          name: 'Excess Internal Temperature',
+          generalDescription: 'Triggers when the internal temperature is too high. This may help preventing '
+          + 'damage to electronics due to overheating.',
```

Comment:
```suggestion
          generalDescription: 'Triggers when the internal temperature is too high. This may help to prevent '
          + 'damage to electronics due to overheating.',
```